packages/ckeditor5-image/src/image/imageediting.js

```diff
@@ -154,22 +153,23 @@ export default class ImageEditing extends Plugin {
 //
 // @private
 // @param {module:engine/view/downcastwriter~DowncastWriter} writer
-// @param {String} imageType
+// @param {String} [imageType] type of created image: 'block' or 'inline'
 // @returns {module:engine/view/containerelement~ContainerElement}
 export function createImageViewElement( writer, imageType ) {
-	const parentName = imageType === 'image' ? 'figure' : 'span';
+	const parentName = imageType === 'block' ? 'figure' : 'span';
+	const className = imageType === 'block' ? 'image' : 'image-inline';
 	const emptyElement = writer.createEmptyElement( 'img' );
-	const figure = writer.createContainerElement( parentName, { class: imageType } );
+	const figure = writer.createContainerElement( parentName, { class: className } );
```

Comment:
Maybe this way it would be clearer?
```javascript
const figure = imageType == 'block' ? 
    writer.createContainerElement( 'figure', { class: 'image' } ) : 
    writer.createContainerElement( 'span', { class: 'image-inline' } );
```